pkg/core/native/policy.go

```diff
@@ -173,6 +181,10 @@ func (p *Policy) Initialize(ic *interop.Context) error {
 		attributeFee:       map[transaction.AttrType]uint32{},
 		blockedAccounts:    make([]util.Uint160, 0),
 	}
+	if p.p2pSigExtensionsEnabled {
+		setIntWithKey(p.ID, ic.DAO, []byte{attributeFeePrefix, byte(transaction.NotaryAssistedT)}, defaultNotaryAssistedFee)
```

Comment:
I'm not sure we'll keep it this way, but we'll see.

Reply:
What's the other option?

Reply:
Leave it at zero by default, as other attributes. Then committee can set it to whatever value needed. But we'll see, it's a part of https://github.com/neo-project/neo/issues/2896.